app/test-e2e/page-objects/page-objects.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function(){
```

Comment:
probably best to name file after page being tested, ie `home.js` or `home-page.js`